src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationOptions.cs

```diff
@@ -202,7 +202,13 @@ public AzureAppConfigurationOptions SetClientFactory(IAzureClientFactory<Configu
         /// The label filter to apply when querying Azure App Configuration for key-values. By default the null label will be used. Built-in label filter options: <see cref="LabelFilter"/>
         /// The characters asterisk (*) and comma (,) are not supported. Backslash (\) character is reserved and must be escaped using another backslash (\).
         /// </param>
-        public AzureAppConfigurationOptions Select(string keyFilter, string labelFilter = LabelFilter.Null)
+        /// <param name="tagsFilter">
+        /// The tag filter to apply when querying Azure App Configuration for key-values. By default no tags will be used.
+        /// Each tag provided must follow the format "tag=value". A key-value will only be returned if its tags contain all tags provided
+        /// in the filter, or if the filter is empty.
```

Comment:
How can users specify tags with null value? Maybe we need to add a default tag filter value like we have LabelFilter.Null today.

Reply:
"tag=" is valid, that's the only way right now. They still have to specify a tag name though so would they have to use the default tag filter value as part of a concatenated/formatted string?

`"tag=" + TagFilter.Null` ?

Reply:
"tag=" means the value is empty string. "tag=\0" means value is null.
I was thinking of something like:
`$"tag={TagValue.Null}`

Reply:
I see, yea I can add that and mention it in the comment here